integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java

```diff
@@ -291,7 +294,7 @@ private void upgradeOperator(String operatorVersion, String externalServiceNameS
 
         assertTrue(upgradeAndVerifyOperator(opNamespace, opParams),
             String.format("Failed to upgrade operator in namespace %s", opNamespace));
-
+        verifyPodsNotRolled(domainNamespace, pods);
```

Comment:
move this check to line 309, that is after making sure the operator image name is modified

Reply:
done

Reply:
done